File: obscure_eye_app/lib/app/views/live_page.dart
import 'package:flutter/material.dart';
import '../theme/cyberpunk_theme.dart';

class LivePage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Live View'),
        centerTitle: true,
        elevation: 0,
      ),
      body: Center(
        child: Column(
          mainAxisAlignment: MainAxisAlignment.center,
          children: [
            Container(
              width: 320,
              height: 200,
              decoration: BoxDecoration(
                color: darkPurple,
                border: Border.all(color: neonLime, width: 3),
                borderRadius: BorderRadius.circular(16),
              ),
              child: Center(
                child: Icon(Icons.videocam, color: neonPurple, size: 64),
              ),
            ),
            SizedBox(height: 24),
            Text(
              'Live camera feed (mock)',
              style: TextStyle(
                color: neonLime,
                fontSize: 18,
                fontWeight: FontWeight.bold,
              ),
            ),
          ],
        ),
      ),
    );
  }
}

File: frontend/obscure-eye-dashboard/src/pages/LivePage.js
import React from 'react';
import { Box, Typography, Card, CardContent } from '@mui/material';
import VideocamIcon from '@mui/icons-material/Videocam';

export default function LivePage() {
  return (
    <Box>
      <Typography variant="h4" color="secondary" gutterBottom>
        Live View
      </Typography>
      <Card sx={{ maxWidth: 480, margin: '32px auto', background: '#2d0036', border: '2px solid #cfff04' }}>
        <CardContent sx={{ display: 'flex', flexDirection: 'column', alignItems: 'center', py: 6 }}>
          <VideocamIcon sx={{ fontSize: 80, color: '#a259f7' }} />
          <Typography variant="h6" color="secondary" sx={{ mt: 2 }}>
            Live camera feed (mock)
          </Typography>
        </CardContent>
      </Card>
    </Box>
  );
}